vignettes/examples.Rmd

```diff
@@ -0,0 +1,350 @@
+---
+title: "Examples of Usage"
```

Comment:
Can we add a more descriptive title? Maybe "Continental Stream Morphology Research empowered by open data"?